module/data/item/weapon.mjs

```diff
@@ -235,12 +235,12 @@ export default class WeaponData extends ItemDataModel.mixin(
     if ( this.parent.labels.damages?.length ) {
       const config = { ...CONFIG.DND5E.damageTypes, ...CONFIG.DND5E.healingTypes };
       context.info.push({ value: this.parent.labels.damages.reduce((str, { formula, damageType }) => {
-        const { label, icon } = config[damageType];
+        const type = config[damageType];
```

Comment:
Probably going to conflict with a similar change on the `activities/ui` branch.

Reply:
Yeah, saw we both fixed the same bug :)